### PR TITLE
KEYCLOAK-6355

### PR DESCRIPTION
**Pull request by patriot1burke**
Please read JIRA for more details on why I did this and what I did.

### CODE REVIEW

**Review by hmlnarik — COMMENTED** [review]
Should this flow support action tokens? If not, from action token perspective this change works, if yes, the https://github.com/patriot1burke/keycloak/blob/d6788a0839eea4811064f6b5af2886af073de7bd/services/src/main/java/org/keycloak/services/resources/LoginActionsService.java#L448 should be updated accordingly.

File: adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java [hunk 1]
```diff
@@ -289,6 +302,86 @@ public void loginManual(PrintStream printer, Reader reader) throws IOException,
...
+     * @throws VerificationException
+     */
+    public boolean loginCommandLine(String redirectUri) throws IOException, ServerRequest.HttpFailure, VerificationException {
+        String authUrl = deployment.getAuthUrl().clone()
+                .queryParam(OAuth2Constants.RESPONSE_TYPE, OAuth2Constants.CODE)
+                .queryParam(OAuth2Constants.CLIENT_ID, deployment.getResourceName())
+                .queryParam(OAuth2Constants.REDIRECT_URI, redirectUri)
+                .queryParam(OAuth2Constants.SCOPE, OAuth2Constants.SCOPE_OPENID)
+                .build().toString();
+        ResteasyClient client = new ResteasyClientBuilder().disableTrustManager().build();
```
Comment (mposolda) [inline adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java]: Doesn't look very safe? Couldn't it reuse the ResteasyClient based on the Apache HTTP Client from the provided KeycloakDeployment, which has all the truststore related stuff setup?

**Review by mposolda — COMMENTED** [review]
Looks fine to me. Besides what Hynek asked, I have just one comment in the KeycloakInstalled class.

File: adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java (hunk 1 above)
Comment (patriot1burke) [reply adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java]: KeycloakInstalled is not part of the server.  This is an experimental feature anyways.

Comment (patriot1burke) [reply adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java]: experemental and optional feature.

**Review by mposolda — APPROVED** [review]
Ok, Thanks

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**